bin/resources/GameIndex.yaml

```
@@ -631,6 +631,8 @@ SCAJ-20101:
 SCAJ-20102:
   name: "Tales of Symphonia"
   region: "NTSC-Unk"
+    gsHWFixes:
+      alignSprite: 1 #fixes vertical bars
```

```suggestion
      alignSprite: 1 # Fixes vertical lines.
```
bars are really just thick lines, which I doubt they are. Change also the other variants to be formatted correctly.